website/docs/plugin/testing/acceptance-tests/plan-checks/output.mdx

```diff
@@ -54,34 +64,142 @@ func TestExpectKnownOutputValue_CheckPlan_Bool(t *testing.T) {
 }
 ```
 
-## Example using `plancheck.ExpectKnownOutputValueAtPath`
+## `ExpectKnownOutputValueAtPath` Plan Check
 
 The [`plancheck.ExpectKnownOutputValueAtPath(address, path, value)`](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-testing/plancheck#ExpectKnownOutputValueAtPath) plan check verifies that a specific output value at a defined path has a known type, and value.
 
+~> **Note**: Prior to Terraform v1.3.0 a planned output is marked as fully unknown if any attribute is unknown.
+
 Refer to [Known Value Checks](/terraform/plugin/testing/acceptance-tests/known-value-checks) for details, and examples of the available [knownvalue.Check](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-testing/knownvalue#Check) types that can be used with the `ExpectKnownOutputValueAtPath` plan check.
 
 ```go
+package example_test
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/knownvalue"
+	"github.com/hashicorp/terraform-plugin-testing/plancheck"
+)
+
 func TestExpectKnownOutputValue_CheckPlan_Bool(t *testing.T) {
 	t.Parallel()
 
-	r.Test(t, r.TestCase{
+	resource.Test(t, resource.TestCase{
+		// Provider definition omitted.
+		Steps: []resource.TestStep{
+			{
+				// Example resource containing a computed boolean attribute named "computed_attribute"
+				Config: `resource "test_resource" "one" {}
+
+                // Generally, it is not necessary to use an output to test a resource attribute,
+                // the resource attribute should be tested directly instead. This is only shown as
+                // an example.
```

Comment:
nit: Might be helpful to describe exactly what "directly" means in this context. (I assume this meant checking state with `statecheck.ExpectKnownValue`)

Reply:
I've updated the wording and add an example in each place where `// Generally, it is not necessary to use an output to test a resource....` appears.